### PR TITLE
Blitz Points

### PR DESCRIPTION
**Pull request by frankfka**

### CODE REVIEW

File: packages/indexer-client/src/types/clientTypes.ts [hunk 1]
```diff
@@ -542,6 +542,41 @@ export type GetIndexerClaimArbMerkleProofsParams =
...
 
+/**
+ * Blitz points
+ */
+
+export interface GetIndexerBlitzPointsParams {
+  address: string;
+}
+
+export interface GetIndexerBlitzPointsResponse {
```
Comment (yruej301) [inline packages/indexer-client/src/types/clientTypes.ts:553]: is it possible to add comments for what each of these fields are?

**Review by yruej301 — COMMENTED** [review]
just realized this is a minor PR. left a comment but would it be possible to also add some sanity checks?

File: packages/indexer-client/src/types/clientTypes.ts (hunk 1 above)
Comment (frankfka) [reply packages/indexer-client/src/types/clientTypes.ts:553]: Done!

File: apps/e2e/src/indexer-client/fullSanity.ts [hunk 2]
```diff
@@ -84,26 +84,43 @@ async function fullSanity(context: RunContext) {
...
     limit: 1,
-    productIds: [1, 2, 3, 4],
+    productIds: [2, 3, 4],
   });
 
   prettyPrint('Market snapshots', marketSnapshots);
 
   const multiProductSnapshots = await client.getMultiProductSnapshots({
-    productIds: [1, 2, 3],
+    productIds: [2, 3],
```
Comment (frankfka) [inline apps/e2e/src/indexer-client/fullSanity.ts:123]: Picking product IDs that work on both blitz & vertex

**Review by yruej301 — APPROVED** [review]
lgtm!

**Pull request merged**